packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -536,15 +571,31 @@ function TokensPanel({
   }, [tokensFromLists, tokensFromUser, newToken, getBalance])
 
   const storeNewToken = async () => {
-    return token.add(newToken).catch((ex: Error) => {
-      let error = 'Token not found on this network.'
+    let error = 'Token not found on this network.'
+    let didSucceed = false
```

Comment:
I understand `didSucceed` but it's uncommon to have var names like this
do you think `isSuccessful` would increase readability?
otherwise just keep the current var name